iTechArt.Api/Controllers/GroceryController.cs

```diff
@@ -89,9 +89,9 @@ public async ValueTask<IActionResult> ImportXMLGroceryFileAsync(IFormFile file)
         /// api route which allows to get all info from db and parse it to the following format
         /// </summary>
         [HttpGet("get_all")]
-        public async Task<IActionResult> ExportGroceryDataAsync()
+        public async Task<IActionResult> ExportGroceryData([FromQuery] int pageIndex)
```

Comment:
Async method doesn`t contain 'Async' postfix.
Check another methods in project

Reply:
Done